test-jest/polar/Pie.spec.tsx

```diff
@@ -352,83 +374,110 @@ describe('<Pie />', () => {
           onMouseEnter={onMouseEnter}
           onMouseLeave={onMouseLeave}
           onClick={onClick}
+          dataKey="cy"
         />
       </Surface>,
     );
-    const se = wrapper.find(Layer).at(3);
-
-    se.simulate('mouseEnter');
-    expect(onMouseEnter.calledOnce).to.equal(true);
-    se.simulate('mouseLeave');
-    expect(onMouseLeave.calledOnce).to.equal(true);
-    se.simulate('click');
-    expect(onClick.calledOnce).to.equal(true);
+
+    const se = container.querySelectorAll('.recharts-layer')[4];
```

Comment:
Please do not use abbreviations.
Did you mean the sector?

Reply:
+1 not sure what `se` is - this was in the previous test but lets expand it out

Reply:
Yeah it was carried over from the original file, I've expanded it out to `sector`.